kairon/api/app/routers/bot/bot.py

```
@@ -1658,3 +1660,60 @@ async def get_slot_actions(
     llm_models = MongoProcessor.get_slot_mapped_actions(current_user.get_bot(), slot_name)
     return Response(data=llm_models)
 
+
+@router.get("/mail/config", response_model=Response)
+async def get_all_mail_configs(
+        current_user: User = Security(Authentication.get_current_user_and_bot, scopes=TESTER_ACCESS)):
```

Ruff (0.7.0)

Do not perform function call `Security` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)